source/js/weekly_navigation.js

```diff
@@ -63,30 +73,129 @@ function populate(key) {
     // Sets all dates for each day
     const days = document.getElementsByClassName('date');
     for (let i = 0; i < 7; i += 1) {
+        // loop through every log
         const tempKey = getDaysKey(window.curDate);
+        const log = getDaysData(tempKey);
+        const allTasks = log.tasks;
+        const allEve = log.events;
+        const day = log.name;
+    
+        window.curDate.setDate(window.curDate.getDate() + 1);
+       
         // eslint-disable-next-line prefer-destructuring
         days[i].innerText = getName(tempKey).split(',')[1];
-        window.curDate.setDate(window.curDate.getDate() + 1);
+    
+
+        const taskDay = log.name.slice(0, 3).toLowerCase();
+        const taskID = ['cal', 'task', taskDay].join('-');
+        const taskBox = document.getElementById(taskID);
+        const id = 'cal-' + day.slice(0, 3).toLowerCase();
```

Comment:
Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
            const id = `cal-${  day.slice(0, 3).toLowerCase()}`;
```